docs/docs/sdk-examples/introduction.mdx

```diff
@@ -0,0 +1,180 @@
+---
+title: Introduction
+sidebar_label: Introduction
+---
+
+You can integrate ZITADEL quickly into your application and be up and running within minutes.
+To achieve your goals as fast as possible, we provide you with SDKs, Example Repositories and Guides.
+
+The SDKs and Integration depend on the framework and language you are using.
+
+import { Tile } from "../../src/components/tile";
+
+### Resources
+
+<div className="tile-wrapper">
+<Tile title="Angular"
+    imageSource="/docs/img/tech/angular.svg"
+    link="/docs/sdk-examples/angular"></Tile>
+<Tile title="Flutter"
+    imageSource="/docs/img/tech/flutter.svg"
+    link="/docs/sdk-examples/flutter"></Tile>
+
+<Tile title="Go"
+    imageSource="/docs/img/tech/golang.svg"
+    link="/docs/sdk-examples/go"></Tile>
+<Tile
+    title="Java"
+    imageSource="/docs/img/tech/java.svg"
+    link="/docs/sdk-examples/java"></Tile>
+<Tile
+    title="NestJS"
+    imageSource="/docs/img/tech/nestjs.svg"
+    link="/docs/sdk-examples/nestjs"></Tile>
+<Tile
+    title="Next.js"
+    imageSource="/docs/img/tech/nextjs.svg"
```

Comment:
The nextJS logo is not vissible on the light theme:

![image](https://github.com/zitadel/zitadel/assets/5411563/e59f9d0a-ba0e-44bc-be7c-5556e56e954b)

Probably you will want to use the same SVG as NextAuth, because that one adepts to the theme correctly:

```suggestion
    imageSource="/docs/img/tech/nextjslight.svg"
```